dali/operators/geometry/affine_transforms/transform_translation.cc

```diff
@@ -76,5 +95,6 @@ class TransformTranslationCPU
 };
 
 DALI_REGISTER_OPERATOR(transforms__Translation, TransformTranslationCPU, CPU);
+DALI_REGISTER_OPERATOR(TransformTranslation, TransformTranslationCPU, CPU);
```

Comment:
Can we make it disappear from the support list?

Reply:
I think you should add `MakeInternal` to the schema.